### ISSUE TITLE
[MU4 Request] Macro to "Past as Cue notes"

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#13570]
It takes a lot of steps to create cue notes in muse score,
the most annoying one, I find, is that you have to
unhide the cue notes in the individual parts,
after hiding them in the score.

It would be extremely helpful to get a tool or macro to
"Past as Cue notes".

At least a command to "Show in parts only" would be super helpful.

**Comment by Tantacrul** [comment]
Good idea.

**Comment by wizofaus** [comment]
Yes please! Just tried adding some cue notes to get this:

![image](https://github.com/user-attachments/assets/5d8405a9-f7bb-4bbe-a3d4-18e2305c1e18)

And it was surprisingly painful, including having to edit the mscx file to put the clef there.

**Comment by iainhallam** [comment]
Sibelius can also paste as a cue that updates the cue notes when the original music changes.

**Comment by shoogle** [comment]
Yes, we shouldn't copy and paste cue notes. Instead we should have something like a line element for cues.

When you add a cue line, MuseScore would ask which instrument's notes you want to display, and then it would show those notes and keep them up to date with the original.

If you make the line longer it would show more notes. If you make it shorter it would show fewer notes.

**Comment by wizofaus** [comment]
That sounds nifty but I suspect it might get tricky in practice - e.g. if you want a queue a violin playing artificial harmonics, and just want to show the sounding pitches (which might not even be shown in the violin part).  Plus you generally do need at least some control over what else other than notes need showing (e.g. I can imagine you might want "pizz." shown because it makes such a huge difference to what you need to listen for, but other directives, particularly expressive ones, are unlikely to be useful).

**Comment by wizofaus** [comment]
BTW there are also parts with cues like this: https://s9.imslp.org/files/imglnks/usimg/e/e4/IMSLP562854-PMLP2072-01_flute.pdf

**Comment by shoogle** [comment]
The cue line could be configured via the Style dialog and/or Properties panel. It could have options to:

- Control the clef, key signature and transposition
- Display sounding pitch for harmonics
- Show detailed playing techniques
- Use a separate cutaway stave (or the user can create a small staff and then add cue lines to it manually)

I imagine most composers would be happy with our defaults. It would mainly be transcribers who would be intested in changing settings to match the cues they see in a particular edition.

**Comment by jeetee** [comment]
If the elements from the cue could be treated somewhat similar to linked parts, It'd allow individual tuning of visibility and sizes or (additional) text markings.

**Comment by rpatters1** [comment]
Is there a planned features per version list somewhere? Cues are a major shortcoming compared to (in particular) Dorico. This needs to be scheduled. At the very least a plugin should be written to automate the steps in the original post.
